plank/pkg/server/endpointer_handler_factory.go

```diff
@@ -0,0 +1,121 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/sirupsen/logrus"
+	"github.com/vmware/transport-go/bus"
+	"github.com/vmware/transport-go/model"
+	"github.com/vmware/transport-go/plank/utils"
+	"net/http"
+	"strings"
+	"time"
+)
+
+// buildEndpointHandler builds a http.HandlerFunc that wraps Transport Bus operations in an HTTP request-response cycle.
+// service channel, request builder and rest bridge timeout are passed as parameters.
+func buildEndpointHandler(svcChannel string, reqBuilder func(w http.ResponseWriter, r *http.Request) model.Request, restBridgeTimeout time.Duration) http.HandlerFunc {
```

Comment:
It might be a good idea to turn `reqBuilder` into a type, rather than having the `func` signature in the method. I know it's internal, but it would make it a little easier to read.